src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/SpecifyCultureForToLowerAndToUpperTests.Fixer.cs

```diff
@@ -0,0 +1,304 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the MIT license.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Xunit;
+using VerifyCS = Test.Utilities.CSharpCodeFixVerifier<
+    Microsoft.NetCore.CSharp.Analyzers.Runtime.CSharpSpecifyCultureForToLowerAndToUpperAnalyzer,
+    Microsoft.NetCore.CSharp.Analyzers.Runtime.CSharpSpecifyCultureForToLowerAndToUpperFixer>;
+using VerifyVB = Test.Utilities.VisualBasicCodeFixVerifier<
+    Microsoft.NetCore.VisualBasic.Analyzers.Runtime.BasicSpecifyCultureForToLowerAndToUpperAnalyzer,
+    Microsoft.NetCore.VisualBasic.Analyzers.Runtime.BasicSpecifyCultureForToLowerAndToUpperFixer>;
+
+namespace Microsoft.NetCore.Analyzers.Runtime.UnitTests
+{
+    public class SpecifyCultureForToLowerAndToUpperFixerTests
+    {
+        [Fact]
+        public async Task CA1311_FixToLowerCSharpAsync_SpecifyCurrentCulture()
+        {
+            const string source = @"
+using System.Globalization;
+
+class C
+{
+    void M()
+    {
+        var a = ""test"";
+        a.[|ToLower|]();
+        a?.[|ToLower|]();
+    }
+}
+";
+
+            const string fixedSource = @"
+using System.Globalization;
+
+class C
+{
+    void M()
+    {
+        var a = ""test"";
+        a.ToLower(CultureInfo.CurrentCulture);
+        a?.ToLower(CultureInfo.CurrentCulture);
+    }
+}
+";
+
+            await new VerifyCS.Test
+            {
+                TestState = { Sources = { source } },
+                FixedState = { Sources = { fixedSource } },
+                CodeActionIndex = 0,
```

Comment:
Is this index express the order of the fixer options? Is it possible to show the `UseInvariantVersion` option before `SpecifyCurrentCulture`?

Reply:
oh I forgot the `SpecifyCurrentCulture` is requested to be first one, never mind